invenio/modules/formatter/format_elements/bfe_edit_record.py

```diff
@@ -19,16 +19,18 @@
 
 """BibFormat element to print a link to BibEdit."""
 
-from invenio.utils.url import create_html_link
 from invenio.base.i18n import gettext_set_language
 from invenio.config import CFG_BASE_URL, CFG_SITE_RECORD
 from invenio.legacy.bibedit.utils import user_can_edit_record_collection
+from invenio.utils.url import create_html_link
 
 
-def format_element(bfo, style):
+def format_element(bfo, style, html_class, link_label="Edit This Record"):
```

Comment:
please provide default value for `html_class` and mark `_('Edit This Record')` for translations